plugins/MauticFocusBundle/Translations/en_US/messages.ini

```
@@ -32,6 +32,7 @@ mautic.focus.form.frequency.hourly = "Once per hour"
 mautic.focus.form.frequency.once = "Once per session"
 mautic.focus.form.frequency.q15m = "Every 15 minutes"
 mautic.focus.form.frequency.q2m = "Every 2 minutes"
+mautic.focus.form.frequency.help = "On 'Once per session', a session interruption occurs when a visitor leaves your website and returns later."
```

```suggestion
mautic.focus.form.frequency.help = "When selecting 'Once per session', the Focus Item shows once for each active browser session. A new session occurs when a visitor leaves your website and returns later."
```

Perhaps this might be a more user-friendly explanation?

---

`When selecting 'Once per session', the Focus Item shows once for each active browser session, which ends when a visitor leaves your website.`

What about this one to make it short? Do you feel the meaning keeps the same?

---

That sounds great! Thanks for the suggestion ;)